src/Trakx.CoinGecko.ApiClient/ClientRegistrations.cs

```diff
@@ -227,5 +228,11 @@ private static void AddClients(this IServiceCollection services, CoinGeckoApiCon
                 .AddHttpMessageHandler<CachedHttpClientHandler>();
 
         }
+
+        private static void ConfigureClient(HttpClient client, CoinGeckoApiConfiguration configuration)
```

Comment:
I think that should be done in `ClientConfigurator.cs`

Reply:
Moved